src/Features/LanguageServer/Protocol/Handler/Commands/ExecuteWorkspaceCommandHandler.cs

```diff
@@ -18,7 +18,7 @@
 namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
     [Shared]
-    [ExportLspMethod(LSP.Methods.WorkspaceExecuteCommandName)]
+    [ExportLspMethod(LSP.Methods.WorkspaceExecuteCommandName, mutatesSolutionState: false)]
```

Comment:
I believe this should be true - applying code action commands will mutate the solution through RunCodeActionsHandler

Reply:
Added a comment to https://github.com/dotnet/roslyn/issues/45427 to do this with that. At the moment there is no code that handles mutation so would hate for this to be forgotten.